### DISCUSSION TITLE
Can we use page level sass along with global sass?

### DISCUSSION THREAD

**Discussion by nitishmittal1990** [vercel/next.js#20898] [Help] [answered]
So, basically what I want to do is 
include global CSS in ```_app.js``` and for the next page with the name ```about.js``` I want to include ```about.scss``` which will have styles specific to about page.

**Comment by jamesmosier** [comment, marked as answer]
The only way to achieve page or component level SASS would be to use CSS (SASS) modules [as shown here](https://nextjs.org/docs/basic-features/built-in-css-support#adding-component-level-css)

```js
// pages/about.js
import styles from '../styles/about.module.scss';

export default function About() {
  return <div className={styles.myStyle}>hi</div>
}
```

```css
// styles.about.module.scss
.myStyle {
  color: red;
}
```

**Reply by bscaspar** [reply]
Would it be ok to put the module file in the same directory as the `about.js` file? Or would that potentially mess with the build, having non-page files in the `/pages` directory?

**Reply by jamesmosier** [reply]
You should only put page `js` files in the `pages` directory, so do not put any of your css module files there. Anything inside `pages` is treated as a new route for your website, so you'll want to be sure to only put actual pages in there 🚀
